guide/002-listen-and-respond.md

```diff
@@ -2,66 +2,66 @@
 
 > This article is a work-in-progress.
 
-Once you start your bot, Telegram sends you updates.
+Once you start your bot, Telegram sends you updates based on each request that is being made to the bot.
 
 > See also: [official reference](https://core.telegram.org/bots/api#getting-updates)
 
-Messages are one type of update, but not the only one. When users send messages, your bot receives a `MessageUpdate`. It looks like: `{ update_id, message }`. When users make callback queries, it receives `CallbackQueryUpdate`, which looks like `{ update_id, callback_query }`.
+Messages are one type of `update` that can be sent, but not the only one. When users send messages, your bot receives a `MessageUpdate`. It looks like this: `{ update_id, message }`. When users make callback queries, it receives `CallbackQueryUpdate`, which looks like this: `{ update_id, callback_query }`.
 
 Telegraf allows you to listen to these updates using a simple syntax:
 
 ```TS (Node)
 bot.on("message", ctx => {
-	// Use ctx.message
+	// Use ctx.message to access the "message" object
 });
 
 bot.on("callback_query", ctx => {
-	// Use ctx.callbackQuery
+	// Usage: ctx.callbackQuery
```

Comment:
This change is not necessary. We're just saying "use ctx.callbackQuery here". See consistency with the above example